provider/terraform/custom_code.rb

```diff
@@ -98,11 +107,22 @@ def config_test
         ))
       end
 
+      def ignore_read_extras
+        @ignore_read_extra ||= []
```

Comment:
If you move this line into the validate then you can get rid of this function altogether